keepalive.go

```diff
@@ -0,0 +1,15 @@
+package grpc
+
+import (
+	"time"
+)
+
+// KeepaliveParameters is used to set keepalive parameters.
+type KeepaliveParameters struct {
+	// After a duration of this time the client pings the server to see if the transport is still alive.
```

Comment:
It seems there should be some mention of the current defaults. Here and below. Try to use language that implies the defaults may change, like "the current default is" and similar.